src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigAdvancedSettingsPage.Designer.cs

```diff
@@ -102,10 +104,34 @@ private void InitializeComponent()
             checkboxMergeAutoStash.ThreeState = true;
             checkboxMergeAutoStash.UseVisualStyleBackColor = true;
             // 
+            // checkBoxReReReEnabled
+            // 
+            checkBoxReReReEnabled.AutoSize = true;
+            checkBoxReReReEnabled.Location = new Point(19, 164);
+            checkBoxReReReEnabled.Name = "checkBoxReReReEnabled";
+            checkBoxReReReEnabled.Size = new Size(275, 19);
+            checkBoxReReReEnabled.TabIndex = 7;
```

Comment:
Could you please confirm the tab order is logical?

Reply:
Yes, it is. The checkboxes of this page have TabIndex 1 to 8 - matching their arrangement.